index.bs

```diff
@@ -1722,7 +1775,8 @@ WebAuthn supports multiple attestation types:
 
 ### Generating an Attestation Object ### {#generating-an-attestation-object}
 
-This section specifies the algorithm for generating an [=attestation object=] for any [=attestation statement format=].
+This section specifies the algorithm for generating an [=attestation object=] (see: [Figure 3](#fig-attStructs)) for any
```

Comment:
I trust you realize that I realize the issue here :)
The `<figure>` and  `<figcaption>` elements are a part of HTML, and AFAICT neither bikeshed, nor HTML, at this time provides auto-numbering for `<figure>`.  Please correct me if I've got this wrong...